src/mcode-pilot/services/outcomes/race_codes.js

```diff
@@ -0,0 +1,942 @@
+    {"level": "2","code": "2052-9","text": "Madagascar"},
+    {"level": "1","code": "2054-5","text": "Black or African American"},
+    {"level": "2","code": "2056-0","text": "Black"},
+    {"level": "2","code": "2058-6","text": "African American"},
+    {"level": "2","code": "2060-2","text": "African"},
+    {"level": "3","code": "2061-0","text": "Botswanan"},
+    {"level": "3","code": "2062-8","text": "Ethiopian"},
+    {"level": "3","code": "2063-6","text": "Liberian"},
+    {"level": "3","code": "2064-4","text": "Namibian"},
+    {"level": "3","code": "2065-1","text": "Nigerian"},
+    {"level": "3","code": "2066-9","text": "Zairean"},
+    {"level": "2","code": "2067-7","text": "Bahamian"},
+    {"level": "2","code": "2068-5","text": "Barbadian"},
+    {"level": "2","code": "2069-3","text": "Dominican"},
+    {"level": "2","code": "2070-1","text": "Dominica Islander"},
+    {"level": "2","code": "2071-9","text": "Haitian"},
+    {"level": "2","code": "2072-7","text": "Jamaican"},
+    {"level": "2","code": "2073-5","text": "Tobagoan"},
+    {"level": "2","code": "2074-3","text": "Trinidadian"},
+    {"level": "2","code": "2075-0","text": "West Indian"},
+    {"level": "1","code": "2076-8","text": "Native Hawaiian or Other Pacific Islander"},
+    {"level": "2","code": "2078-4","text": "Polynesian"},
+    {"level": "3","code": "2079-2","text": "Native Hawaiian"},
+    {"level": "3","code": "2080-0","text": "Samoan"},
+    {"level": "3","code": "2081-8","text": "Tahitian"},
+    {"level": "3","code": "2082-6","text": "Tongan"},
+    {"level": "3","code": "2083-4","text": "Tokelauan"},
+    {"level": "2","code": "2085-9","text": "Micronesian"},
+    {"level": "3","code": "2086-7","text": "Guamanian or Chamorro"},
+    {"level": "3","code": "2087-5","text": "Guamanian"},
+    {"level": "3","code": "2088-3","text": "Chamorro"},
+    {"level": "3","code": "2089-1","text": "Mariana Islander"},
+    {"level": "3","code": "2090-9","text": "Marshallese"},
+    {"level": "3","code": "2091-7","text": "Palauan"},
+    {"level": "3","code": "2092-5","text": "Carolinian"},
+    {"level": "3","code": "2093-3","text": "Kosraean"},
+    {"level": "3","code": "2094-1","text": "Pohnpeian"},
+    {"level": "3","code": "2095-8","text": "Saipanese"},
+    {"level": "3","code": "2096-6","text": "Kiribati"},
+    {"level": "3","code": "2097-4","text": "Chuukese"},
+    {"level": "3","code": "2098-2","text": "Yapese"},
+    {"level": "2","code": "2100-6","text": "Melanesian"},
+    {"level": "3","code": "2101-4","text": "Fijian"},
+    {"level": "3","code": "2102-2","text": "Papua New Guinean"},
+    {"level": "3","code": "2103-0","text": "Solomon Islander"},
+    {"level": "3","code": "2104-8","text": "New Hebrides"},
+    {"level": "2","code": "2500-7","text": "Other Pacific Islander Note that this term remains in the table for completeness, even though within HL7, the notion of Other code is deprecated."},
```

Comment:
Looks like a little extra got copy & pasted on this line.

Reply:
Yeah, that was included in the list I copied the codes from.  Removed.